View_Assist_custom_sentences/Play_Music_with_Music_Assistant/blueprint-playmusicwithmusicassistant.yaml

```diff
@@ -24,153 +24,262 @@ blueprint:
       description: The View Assist dashboard view to use for displaying music information (example /dashboard-viewassist/music)
       default: "/dashboard-viewassist/music"
 alias: View Assist - Play Music with Music Assistant
-description: ""
-trigger:
-  - platform: conversation
-    command:
+description: "Control volume, playback and play music using music assistant"
+triggers:
+  - command:
       - play the artist {artist}
-      - play some {artist}
-    id: playartist
-  - platform: conversation
-    command:
+      - play some {artist} music
+      - play some music by {artist}
+    id: artist
+    trigger: conversation
+  - command:
       - start [the] {playlist} playlist
     id: playlist
-  - platform: conversation
-    command:
+    trigger: conversation
+  - command:
       - play {song} by {artist}
-    id: playsong
-  - platform: conversation
-    command:
+    id: track
+    trigger: conversation
+  - command:
       - queue {song} by {artist}
       - cue {song} by {artist}
-    id: cuesong
-condition: []
-action:
+    id: cuetrack
+    trigger: conversation
+  - trigger: conversation
+    command:
+      - turn up the volume
+      - turn up the music
+      - turn up the music volume
+      - increase the music volume
+    id: volume_up
+  - trigger: conversation
+    command:
+      - turn down the volume
+      - turn down the music
+      - turn down the music volume
+      - decrease the music volume
+    id: volume_down
+  - trigger: conversation
+    command:
+      - pause the music
+      - resume the music
+    id: pause
+  - trigger: conversation
+    command:
+      - next song
+      - skip this song
+      - skip to next song
+    id: next
+conditions: []
+actions:
+  - variables:
+      target_musicplayer_device: media_player.picoreplayer
   - choose:
       - conditions:
           - condition: trigger
             id:
-              - playartist
+              - artist
         sequence:
-          - service: python_script.set_state
-            data:
-              entity_id: "{{ target_satellite_device }}"
-              mode: music
-              title: ""
-          - service: browser_mod.navigate
-            metadata: {}
-            data:
-              path: "{{ view_path }}"
-            target:
-              device_id: "{{ target_display_device }}"        
-          - service: music_assistant.search
-            data:
+          - data:
               limit: 1
               config_entry_id: "{{ ma_instance }}"
               media_type:
-                - artist
+                - "{{trigger.id}}"
```

Comment:
I'm not sure that this is correct.  Why should this not be artist?

Reply:
This is correct and does work. It basically refuses the trigger ID to pass the right context to MA. I did this so it's easier to add other searches in the future.

Reply:
I see what you are doing.  I think these are unnecessary changes.  Please revert all "{{trigger.id}}" changes back to original.